packages/push-macros/src/push_state/printing/generate_builder.rs

```diff
@@ -333,13 +333,14 @@ pub fn generate_builder(
             /// # Arguments
             /// - `program` - The program you wish to execute
             #[must_use]
-            pub fn with_program<P>(mut self, program: P)
+            pub fn with_program<P, I>(mut self, program: P)
                 -> ::std::result::Result<#builder_name<#utilities_mod_ident::WithSizeAndData, #(#stack_generics),*>, ::push::push_vm::stack::StackError>
             where
-                P: ::std::iter::IntoIterator<Item = ::push::instruction::PushInstruction>,
+                P: ::std::iter::IntoIterator<Item = I>,
                 <P as ::std::iter::IntoIterator>::IntoIter: ::std::iter::DoubleEndedIterator + ::std::iter::ExactSizeIterator,
+                I: ::std::convert::Into<::push::push_vm::program::PushProgram>
```

Comment:
It won't let me apply this suggestion in the web UI because there are deleted lines. I think we'll want a separate commit for this. Should I let this go and we'll fix it in a stream, or should I do the change in VSCode and commit it?

Reply:
I don't care tbh, we can do whatever you prefer. All the stuff here was minor anyways.